packages/jest-config/src/normalize.js

```diff
@@ -367,6 +372,17 @@ export default function normalize(options: InitialOptions, argv: Argv) {
     ),
   );
 
```

Comment:
I could push the value of `setupTestFrameworkScriptFile` into the `setupFilesAfterEnv` array here.